llvmlite/ir/values.py

```diff
@@ -649,6 +649,48 @@ class ArgumentAttributes(AttributeSet):
                         'nocapture', 'nonnull', 'returned', 'signext',
                         'sret', 'zeroext'])
 
+    def __init__(self):
+        self._align = 0
+        self._dereferenceable = 0
+        self._dereferenceable_or_null = 0
+
+    @property
+    def align(self):
+        return self._align
+
+    @align.setter
+    def align(self, val):
+        assert isinstance(val, int) and val >= 0
```

Comment:
Can you replace `int` to `six.integer_types` so that it works with python2 long?  (See https://github.com/numba/llvmlite/pull/236/files#diff-daa10273fc8d59dae221475ac5306b12R409 for reference)